packages/client/src/components/AppNavBar.tsx

```diff
@@ -0,0 +1,69 @@
+import MenuIcon from '@mui/icons-material/Menu';
+import ShoppingBasketIcon from '@mui/icons-material/ShoppingBasketOutlined';
+import { AppBar, Box, Button, Fab, IconButton, Link, Menu, MenuItem, Toolbar, useTheme } from '@mui/material';
+import { useState } from 'react';
+
+import logo from '../assets/logo.svg';
+
+const PAGES = ['O nas', 'Kontakt', 'Dostawa'] as const;
+
+const AppNavBar = () => {
+  const theme = useTheme();
+
+  const [menuAnchorElem, setMenuAnchorElem] = useState<null | HTMLElement>(null);
+
+  const handleMenuOpen = (event: React.MouseEvent<HTMLElement>) => {
+    setMenuAnchorElem(event.currentTarget);
+  };
+
+  const handleMenuClosed = () => {
+    setMenuAnchorElem(null);
+  };
+
+  return (
+    <AppBar sx={{ background: '#1B5E2014' }} position='fixed'>
```

Comment:
Zastanawia mnie czy nie lepiej dodać kolor `#edf2ed` do naszego `theme` zamiast korzystać z `primary: main` z przezroczystością, skoro i tak wtedy trzeba hardcodować ten kolor. Co o tym myślisz? Zmieniłybyśmy wtedy to w Figmie